controllers/remote/cluster_cache_tracker.go

```diff
@@ -139,6 +153,13 @@ func setDefaultOptions(opts *ClusterCacheTrackerOptions) {
 			&corev1.Secret{},
 		}
 	}
+
+	if opts.ClientQPS == 0 {
+		opts.ClientQPS = 20
+	}
+	if opts.ClientBurst == 0 {
+		opts.ClientBurst = 30
+	}
```

Comment:
I interpret that this case will be occurred when the someone specifies ClientQPS or ClientBurst to 0. If it fills out with default value arbitrarily, someone may think the 0 is valid value. So I think this case should return error.

Reply:
Nope, this cases get in two cases:

* when an implementer does not set `ClientQPS` and/or `ClientBurst` on the opts `ClusterCacheTrackerOptions`
* when an implementer sets them, but the user passes in `0` as value

IMHO its fine as is.

Reply:
Hi @chrischdi 

Thank you for teaching me 👍 

> when an implementer sets them, but the user passes in 0 as value

I just worried whether it's better that we let implementer know about the information when the actual value of ClientQPS and/or ClientBurst is different from args implementer sets. But I think it is fine too. Thanks.

Reply:
The problem here is that we don't know if someone set the fields explicitly to 0, are they are just 0 because 0 is the "zero value" of this fields.

I don't want to force everyone to set these values, which would happen if we always return an error if the fields are 0

Reply:
Hi @sbueringer 
I understood it and I agree with your opinion. Surely, this field is option as it is named and it should not force everyone to set these.
Thanks.